_posts/2020-03-11-guide-for-unmaintained-open-source-projects.markdown

```diff
@@ -0,0 +1,177 @@
+- https://medium.com/the-node-js-collection/healthy-open-source-967fa8be7951
+
+### For OSS Users and Contributors
+
+What can you do as someone who actively uses or contributes to the OSS project? 
+For example: What can you do when you **really really** need to have support for 
+Rails 6.0 for your application but this library is blocking you?
+
+#### Best Judgement for Assessing Abandonment 
+
+Use your best judgment to determine whether the project is truly abandoned or 
+just feature complete. 
+
+> There are many projects out there that are not being actively maintained because 
+they do one little thing really well. Think about these projects as the outliers
+because they are feature complete and have no bugs.
+
+Here are some things to consider when judging the state of an OSS project:
+
+- When was the last time there was a commit to the `master` branch? 
+- When was the last time there was a commit to a non-master branch? 
+- When was the last time there was a comment in an issue by a maintainer?
+- When was the last time there was a code review from one of the maintainers?
+- Is the test suite still passing if you submit one small change to the project?
+- When was the last version of the library published? (Example: for a Ruby gem, 
+you can check in https://rubygems.org)
+- When was the last time an issue or pull request was closed by a maintainer?
+
+#### Inquire
+
+If someone hasn't done it already, you can submit an issue to the project asking 
+about the state of things. 
+
+Before you submit your inquiry, you might want to submit a pull request with a 
+small improvement to the project. If the change is small enough and it improves 
+things, then it should be easy to review or close. Make sure that the test suite 
+passes in CI.
+
+You can start your inquiry by saying that you appreciate their dedication and 
+thank them for their hard work over the years, then you can ask if they're 
+looking for help maintaining the project.
+
+**Do not** demand an answer from the maintainers. Maintainers don't owe you much,
+so you will be lucky if you get a response to a nice request. So, 
+**please be nice**.
+
+You don't need to offer yourself as a potential maintainer for the project, 
+maybe there are other people out there who are in a better position to 
+take over.
+
+If you don't get any response for a month, then you can move on to the next 
+section.
+
+#### Start your own fork
+
+You could start your own OSS project based on a fork. Before you do that, make 
+sure that you check the contribution graph for the project. There might be 
+someone else updating a fork that already has all the changes you need.
+
+> Fun fact: [Forking a project used to be considered a "bad word"](https://www.bacula.org/why-forking-is-bad/) 
+> but I think the rise of GitHub turned that around. 
+
+This is what [Solidus](https://Solidus.io) did with [Spree](https://spree.com).
```

Comment:
Did `Solidus` was started because `Spree `was not maintained? or because different ideas on the vision of the project?

Reply:
@bronzdoc Because there was a gap in maintenance. Spree's maintenance was abandoned for months.